scripts/manage-service.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+
+ACTION=""
+if [ -z $1 ]; then
+    echo "Missing paramter, run :
+     $0 deploy or
+     $0 delete"
+     exit 1
+fi
+if [[ "$1" == "deploy" ]]; then
```

Comment:
Every other parameter than `deploy` means delete. Print help message on parameter absence or not deploy or delete. There is no need " in case of `FOO=bar`.

Reply:
Discussed with Brandon about this and we decided to use params for each case to make it consistent with the name of the script "manage-services.sh"

Reply:
I agree if there are options we should probably be explicit as to what operation we're doing.

Reply:
@edcdavid The params are OK. I suggest to handle unknown parameters as errors.

Reply:
fixed